### DISCUSSION TITLE
No items found to extract, review your harvest plan.

### DISCUSSION THREAD

**Discussion by owen-soak** [GetDKAN/dkan#4325] [Migration help]
Good day, 

i've been attempting to configure a harvest on my local site so initially i chose an example from the site (https://dkan.readthedocs.io/en/latest/user-guide/guide_harvest.html) but i get the below error each. Am i missing something?

```In HarvestService.php line 215:
                                                        
  No items found to extract, review your harvest plan. 
```
Command i'm running
`drush dkan:harvest:register --identifier=sec_content --extract-uri=https://www.sec.gov/data.json`
`drush dkan:harvest:run sec_content`

**Comment by janette** [comment]
@owen-soak the identifiers for your datasets need to be uuids.

**Reply by owen-soak** [reply]
@janette So if i wanted to bulk harvest would this be possible? Thank you for your reply!

**Reply by janette** [reply]
It is possible, you would need to build a custom transform to strip the "https://www.sec.gov/node/" from the identifiers.

**Reply by janette** [reply]
See https://github.com/GetDKAN/socrata_harvest/blob/master/src/Harvest/Transform/Socrata.php#L17
